src/web-ui/src/public/CategoryDetail.vue

```diff
@@ -111,7 +122,9 @@ export default {
         intermediate = data
       }
 
-      if (this.personalizeUserID && intermediate.length > 0) {
+      const shouldPersonalise = false;
```

Comment:
Assuming this was added for testing? Please remove.

Reply:
Yep no idea how I let that slip through, removing now